src/module/actor/npc/sheet.ts

```diff
@@ -497,15 +495,6 @@ export class NPCSheetPF2e<TActor extends NPCPF2e> extends CreatureSheetPF2e<TAct
         }
     }
 
-    private async onClickToChat(event: JQuery.ClickEvent): Promise<void> {
-        event.preventDefault();
-
-        const itemId = $(event.currentTarget).parents(".item").attr("data-item-id") ?? "";
-        const item = this.actor.items.get(itemId, { strict: true });
-        if (item instanceof PhysicalItemPF2e && !item.isIdentified) return;
-        await item.toChat(event);
-    }
-
```

Comment:
This has some unique behavior to it that'd be lost: not sure of the wisdom of that behavior, but removing it doesn't seem germane to the PR.

Reply:
I added that behavior to the new one, other than the throwing an error if it doesn't exist bit.

Reply:
I think it's fine that it throw an error: if the item ID is bogus, a much bigger problem is present.